Shapes/Circle.js

```diff
@@ -9,13 +9,19 @@ var {
   }
 } = React;
 
+var makeCirclePath = function(x, y, radius, direction) {
+  var arcMethod = null;
```

Comment:
Please write this in a short `if` statement instead `arcMethod = (direction === 'counterClockwise' ? 'counterArc' : 'arc')`.